.github/workflows/push-image.yml

```diff
@@ -0,0 +1,37 @@
+name: Image-Build-And-Push
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build-and-push:
```

Comment:
We should stop the jobs from `forked` repository, and the reason just like what we did on [Karmada repo](https://github.com/karmada-io/karmada/blob/808e487a2d977e9be9c3d38747d3f11f01826146/.github/workflows/image.yml#L9-L12).

Reply:
Thanks, I will revise it.